### PR TITLE
Add lit

### PR DESCRIPTION
**Pull request by isuruf**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge/staged-recipes ready for review.

I tried adding this as an output in llvmdev and there were multiple issues there. conda-build wasn't activating the environment for llvmdev and I added a workaround, but that worked only for non-windows platforms. Also, it added 3 python builds, but different python versions are not needed.

Until conda-build is fixed, I'm proposing we add this separately and archive the feedstock when the multiple outputs recipe is added.

File: recipes/lit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - lit
+
+about:
+  home: https://llvm.org
+  license: NCSA
+  license_file: LICENSE.txt
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (beckermr) [inline recipes/lit/meta.yaml]: This summary is from the example.

Comment (isuruf) [reply recipes/lit/meta.yaml]: Fixed

**Comment by isuruf** [comment]
Thanks

**Pull request merged**